### PR TITLE
fix: tagger workflows not working

### PR DESCRIPTION
**Pull request by MaikoTan**

### CODE REVIEW

**Review by shigma — APPROVED** [review]

File: .github/workflows/tagger.yml [hunk 1]
```diff
@@ -17,27 +17,22 @@ jobs:
...
+        run: echo "TAG=v$(node -p "require('./packages/core/package.json').version")" >> $GITHUB_OUTPUT
 
       - name: Check if tag exists
         id: check_tag
         run: |
-          if [[ "${{ steps.check_tags.outputs.TAGS }}" == *"${{ steps.check_version.outputs.VERSION }}"* ]]; then
-            echo "TAG_EXISTS=true" >> $GITHUB_OUTPUT
-          else
+          git fetch --tags
+          if [ -z "$(git tag -l | grep -w ${{ steps.check_version.outputs.TAG }})" ]; then
```
Comment (shigma) [inline .github/workflows/tagger.yml:26]: Edge case: a.b.c matches a.b.c-alpha.0?

Comment (MaikoTan) [reply .github/workflows/tagger.yml:26]: You're right, I make a simple test:

```bash
> echo -e "a.b.c-alpha.0" | grep -w a.b.c
a.b.c-alpha.0
```

But I don't know how to construct a full match in `grep`. Anyone aware?

Comment (shigma) [reply .github/workflows/tagger.yml:26]: `^` & `$`?

Comment (MaikoTan) [reply .github/workflows/tagger.yml:26]: Also `$` should be escaped as `\$`, yes.
